### PR TITLE
D4s-v5 recommendation

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

**Comment by aevesdocker** [comment]
@christophermclellan I've amended the sentence and added info about Docker Business support

File: desktop/nested-virtualization.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ You may also need to [configure some network settings](https://www.vembu.com/blo
 
 Nested virtualization is supported by Microsoft for running Hyper-V inside a VM running on a Hyper-V host, in Azure or on-prem (Hyper-V on Hyper-V).
 
-For Azure virtual machines, [check that the VM size chosen supports nested virtualization](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes). Microsoft provides [a helpful list on Azure VM sizes](https://docs.microsoft.com/en-us/azure/virtual-machines/acu) and highlights the sizes that currently support nested virtualization.
+For Azure virtual machines, [check that the VM size chosen supports nested virtualization](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes). Microsoft provides [a helpful list on Azure VM sizes](https://docs.microsoft.com/en-us/azure/virtual-machines/acu) and highlights the sizes that currently support nested virtualization. Docker recommends using D4s_v5 machines. Internal testing confirms this is a good option for performance.
```
[the remark this answers is not in the document]
Comment (christophermclellan) [reply desktop/nested-virtualization.md:43]: Hey @aevesdocker, sorry for delay ! Could we go for `Docker uses D4s_v5 machines for internal testing and would recommend this specification or above for optimising the performance of Docker Desktop.` (or something of that nature)

[the remark this answers is not in the document]
Comment (aevesdocker) [reply desktop/nested-virtualization.md:43]: No problem @christophermclellan. Have amended accordingly.

**Review by dockertopia — APPROVED** [review]
LGTM

**Pull request merged**